examples/README.md

```diff
@@ -0,0 +1,94 @@
+# Examples
+
+this directory contains examples for a few features of the autograder
+
+## Assessments
+
+The Hello folder and Hello.tar contain various examples of different styles of Assessments
+
+
+## Importing Assessments
+
+whether you are importing one or more assessments the procedure is the same
+
+multiple assessment import starter.tar is the starter file for importing one or more assessments
+
+
+```bash
+multiple assessment import starter.tar
+├── assessment1
+│   ├── handin
+│   ├── assessment1.yml
+│   ├── autograde.tar
+│   └── autograde-Makefile
+└── assessment2
+    ├── handin
+    └── assessment1.yml
+
+```
+
+the top level must consist of folders named the same as what you will be naming the course (url name not display name) 
```

Comment:
I don't understand this; so the structure for the tarball would actually be something like:

```
cse220 -,
              |--- assessment1
        |    |- assessment1.yml
        :    : 
```

Reply:
yeah it appears the tree didnt render exactly how it should have but in the one you posted if the tar ball is named cse220 then the folder assessment1 will be the name of the assessment. assessments have 2 names display and db. the folder name maps to the db assessment name

Reply:
Let me clarify; suppose that my course is autograder.tld/courses/cse220/.

Must the tarball be named `cse220.tar` and _everything in the tarball_ contained in a directory named `cse220/`?

If so, I think this is a mistake.  If not, I don't understand the documentation.